docs/setup-ci.md

```diff
@@ -0,0 +1,102 @@
+```
+$fileContentBytes = get-content '<Path-to-Pfx>' -Encoding Byte
+[System.Convert]::ToBase64String($fileContentBytes) | Out-File pfx-encoded-bytes.txt
+```
+in PowerShell. Then upload the contents of `pfx-encoded-bytes.txt` as your secret.
+
+### Accessing Certificate Data from Pipeline
+See the [Xaml-Islands-Samples](https://github.com/microsoft/Xaml-Islands-Samples/blob/master/.github/workflows/CPP-CI.yml) repository for an example of a pipeline which uses Github Secrets.
+See the [react-native-gallery](https://github.com/microsoft/react-native-gallery/blob/main/ci.yml) repository for an example of a pipeline which uses Azure DevOps Secure Files.
+See the react-native-windows repository for an example of a pipeline which uses Azure Key Vault:
+[Setup of Pfx](https://github.com/microsoft/react-native-windows/blob/main/.ado/templates/setup-certificate.yml)
+[Removal of Pfx](https://github.com/microsoft/react-native-windows/blob/main/.ado/templates/cleanup-certificate.yml)
+[Signed RNW App Build](https://github.com/microsoft/react-native-windows/blob/main/.ado/templates/run-windows-with-certificates.yml)
+
+For Azure Secure Files, retrieving the secret will download the .pfx itself. The remaining two methods will give you the encoded string. Thus, you'll have to decode the secret, and save it to a .pfx file before it can be used to sign.  See the Xaml-Islands-Samples repository or the react-native-windows repository for the powershell commands to decode and generate .pfx files within a pipeline.
+	
+Make sure to delete your .pfx from the pipeline once you've finished using it.
+
+## FAQ's
+### I'm ready to do a signed app build. How do I tell msbuild to use my certificate?
+You can tell msbuild what .pfx you want to use to sign your app using the `PackageCertificateKeyFile` msbuild property. This property expects the file path to the .pfx you want to use. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/jobs/playground.yml#L114) for an example of this using VSBuild. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/templates/run-windows-with-certificates.yml#L48) for an example of this using the RNW CLI.
+
+### I have a pipeline that runs on forks of my repository (i.e. when a PR is being made). Can I access my certificate data from this pipeline?
+No. When data is securely stored through Github Secrets, Azure DevOps Secure Files, or Azure Key Vault, it can only be run from pipelines that are executing on branches of the original repository. The data cannot be accessed from pipelines running code from repository forks. Why? Because if the data was able to be accessed, someone could manipulate the pipeline source code within their fork to retrieve the data, leaving it unsecured.
```

Comment:
```suggestion
No. When data is securely stored through GitHub Secrets, Azure DevOps Secure Files, or Azure Key Vault, it can only be run from pipelines that are executing on branches of the original repository. The data cannot be accessed from pipelines running code from repository forks. Why? Because if the data was able to be accessed, someone could manipulate the pipeline source code within their fork to retrieve the data, leaving it unsecured.
```